storage/merge_iterator.go

```diff
@@ -0,0 +1,143 @@
+package storage
+
+import (
+	"bytes"
+	"container/heap"
+)
+
+type iterHeap []Iterator
+
+func (h iterHeap) Len() int {
+	return len(h)
+}
+
+func (h iterHeap) Less(i, j int) bool {
```

Comment:
The sort interface is part of the [`heap` package's interface](https://golang.org/pkg/container/heap/#Interface), so it would be used when doing the heap operations.

Reply:
ah, thanks for the hint, didn't know